pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -0,0 +1,475 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: rh-advisories
+  labels:
+    app.kubernetes.io/version: "0.1.0"
```

Comment:
Would it make sense to start with the same version that the original pipeline had (plus one) and add a release note saying that it's a clone of it and what you added on top of that? That would also help me with the review, because right now I don't know what was added. (But I guess I better do a git checkout  and diff anyway.)

Reply:
I am not sure I agree with having a new pipeline having a not new version

Reply:
Hmm, ok, I don't mind starting 0.1.0, but it would be nice to capture somewhere that it was copied from the other pipeline (and which version). But on the other hand, if the intention is that eventually this will be applied to the rh-push-to-registry-redhat-io pipeline, then I guess we don't need to worry about this much.

Reply:
I wrote that in the description in the README. I can add what version it comes from though, that is a good idea. When I rebase I will add that